index.html

```diff
@@ -1,251 +1,342 @@
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="./assets/css/style.css">
-
-</head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+  </head>
 
-<body>
+  <body>
     <header>
-        <nav>
-            <ul class="mobile">
-                <li><a>Welcome</a></li>
-                <li>
-                    <a class="menu"><i class="fa-solid fa-bars"></i></a>
-                </li>
-            </ul>
-            <ul class="desktop">
-                <li><a href="/">Portfolio</a></li>
-                <li><a href="/">About</a></li>
-                <li><a href="/">Contact</a></li>
-            </ul>
-        </nav>
-        <section class="head-line">
-            <section class="title">
-                <h1>Hey there. I’m David</h1>
-                <h2 class="title2">I’m a software developer</h2>
-            </section>
-            <section class="subtitle">
-                <p> I can help you build a product , feature or website Look through some of my work and experience! If
-                    you
-                    like what you see and have a project you need coded,
-                    don’t hesitate to contact me.</p>
-            </section>
-            <div class="network">
-                <ul>
-                    <li><a href="https://github.com/wdavidcch"><img src="./assets/img/github.png" alt="icon"></a></li>
-                    <li><a href="https://www.linkedin.com/in/williams-colmenares-989a6b151"><img src="./assets/img/linkedin.png" alt="icon"></a></li>
-                    <li><a href="/"><img src="./assets/img/peaceful.png" alt="icon"></a></li>
-                    <li><a href="https://twitter.com/wdavidcch"><img src="./assets/img/twitter.png" alt="icon"></a></li>
-                    <li><a href="/"><img src="./assets/img/medium.png" alt="icon"></a></li>
-                </ul>
-            </div>
-            <div class="scroll-down">
-                <a href="/"><i class="fa-solid fa-circle-chevron-down"></i></a>
-            </div>
+      <nav>
+        <ul class="desktop">
+          <li>
+            <a class="x"><i class="fa-solid fa-x"></i></a>
+          </li>
+          <li><a href="#portfolio" class="link">Portfolio</a></li>
+          <li><a href="#about" class="link2">About</a></li>
+          <li><a href="#contact" class="link3">Contact</a></li>
+        </ul>
+        <ul class="mobile">
+          <li><a>Welcome</a></li>
+          <li>
+            <a class="menu"><i class="fa-solid fa-bars"></i></a>
+          </li>
```

Comment:
- [ **OPTIONAL** ] Good job with all the mobile menu features working as expected. But I think it would be better if you add on **Hamburger(menu-bars)** and **Close-menu** a `cursor pointer property` so it can help every user know that they are clickable.

Reply:
Got it! thank you so much